README.md

```diff
@@ -29,30 +28,41 @@ If you are not interested in building yourself, the pre-built files are availabl
 3. Power on the device, wait about a minute for it to be ready.
 4. Open a web browser, navigate to http://192.168.1.1 and wait for the wizard to come up.
 5. Click *exactly* inside the radio button to confirm the terms and conditions, then abort the wizard.
-6. You should then be greeted by the login screen, the stock password is "admin". (This step may be required for Linksys E8450 only)
-7. Navigate to __Administration__ -> __Firmware Upgrade__
+6. You should then be greeted by the login screen, the stock password is "admin".
+7. Navigate to __Administration__ -> __Firmware Upgrade__.
 8. Upload `openwrt-mediatek-mt7622-linksys_e8450-ubi-initramfs-recovery-installer.itb`.
-9. Wait for a few minutes, the OpenWrt recovery image should come up.
-10. Login and navigate to __System__, save a copy of each of the `mtdblock`.
-11. Upload `openwrt-mediatek-mt7622-linksys_e8450-ubi-squashfs-sysupgrade.itb`.
+9. Wait for a minute, the OpenWrt recovery image should come up.
+9. Navigate to __System__ -> __Backup / Flash Firmware__.
+10. Upload `openwrt-mediatek-mt7622-linksys_e8450-ubi-squashfs-sysupgrade.itb`.
 12. The device will reboot, you may proceed to setup OpenWrt.
 
-## Upgrading to the Latest OpenWrt Snapshot Release.
+## Upgrading to the latest OpenWrt snapshot 
 
 **WARNING**
 SNAPSHOT RELEASES ARE LARGELY UNTESTED!
 PROCEED AT YOUR OWN RISK!
 
-Attended Sysupgrade (auc) is included since version 0.6, all you need to do is to connect the router to the Internet and run `auc` from the web interface.
+1. Backup the original/vendor bootchain
 
-## To Enter Recovery Mode under OpenWrt
+   Connect to the device via SSH and enter the following commands:
+
+   ```
+   mkdir /tmp/boot_backup
+   mount -t ubifs /dev/ubi0_3 /tmp/boot_backup
+   ```
+
+   Then, copy the files under */tmp/boot_backup* using *scp* to your host. These files are needed in case you want to restore the original/vendor firmware. They can also be used in emergency case for reflashing via [JTAG](https://openwrt.org/toh/linksys/e8450#jtag)
+
+2. `auc` (Attended Sysupgrade) and `luci-app-attendedsysupgrade` are included since version 0.6. Simply navigate to __System__ -> __Attended Sysupgrade__ and proceed accordingly.
```

Comment:
```suggestion
2. `auc` and `luci-app-attendedsysupgrade` are included since version 0.6. 
  * To update using the LuCI web interface, navigate to __System__ -> __Attended Sysupgrade__ and proceed accordingly.
  * To update using the command line use `auc`, e.g. when connecting to the router via SSH.
```

`auc` is the "**a**ttended sys**U**pgrade **c**ommand-line client".
`luci-app-attendedsysupgrade` is a LuCI web-interface based client using the same web service.
(`/etc/config/attendedsysupgrade` is the configuration)

Reply:
:+1: